src/Auditor.js

```diff
@@ -0,0 +1,67 @@
+const returnAuditOptions = (updateType, item, data_type) => {
```

Comment:
Please use lower case file names.  There's a lot of repetition in this file, let's see if we can clean it up.